cmd/nvidia-dra-controller/imex.go

```diff
@@ -312,3 +310,33 @@ func (m *ImexManager) cleanupResourceSlices() error {
 
 	return nil
 }
+
+// getOffset set the offset where an imex domain's channels should start counting from.
+func (i *imexDomainOffsets) getOffset(addedDomainsCh string) (int, error) {
```

Comment:
As a map you don't need to have a pointer reference -- it will update by reference anyway